# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/argon2/hotfix-pc-path.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- phc-winner-argon2-20190702/Makefile.vanilla	2024-06-28 02:19:57.296480642 +0300
+++ phc-winner-argon2-20190702/Makefile	2024-06-28 02:20:40.826478133 +0300
@@ -143,7 +143,7 @@
 # default for Debian/Ubuntu x86_64
 BINARY_REL ?= bin
 INCLUDE_REL ?= include
-LIBRARY_REL ?= lib/x86_64-linux-gnu
+LIBRARY_REL ?= lib
 PKGCONFIG_REL ?= $(LIBRARY_REL)
 
 else
